engine/player/unique_gear.cpp

```diff
@@ -1828,6 +1830,24 @@ void item::infallible_tracking_charm( special_effect_t& effect )
   new dbc_proc_callback_t( effect.item, effect );
 }
 
+void item::witherbarks_branch( special_effect_t& effect )
+{
+  auto data_spell = effect.player->find_spell( 429257 );
+  auto stat_buff  = make_buff<stat_buff_t>( effect.player, "aqueous_dowsing", effect.player->find_spell( 429257 ) )
+    ->add_stat( STAT_MASTERY_RATING, data_spell->effectN( 1 ).average( effect.item ) );
+
+  effect.custom_buff = make_buff( effect.player, "aqueous_dowsing_driver", effect.driver() )
+    ->set_quiet( true )
+    // assumes an average of 1 seconds per ball
+    // TODO: make this confirgurable?
+    ->set_cooldown( timespan_t::from_seconds(1) )
+    ->set_tick_callback( [ stat_buff ]( buff_t*, int, timespan_t ) {
+      stat_buff->trigger();
+    } );
```

Comment:
for this effect, if youre attempting to make it trigger a stack every 1 second, should use `set_period( 1_s )`, rather than set cooldown. right now, theres no periodic effect on the driver, so, the tick callback is likely never called.